.setup/bin/setup_sample_courses.py

```diff
@@ -962,18 +963,25 @@ def create(self):
                         if gradeable.grade_released_date < NOW or (random.random() < 0.5 and (submitted or gradeable.type !=0)):
                             status = 1 if gradeable.type != 0 or submitted else 0
                             print("Inserting {} for {}...".format(gradeable.id, user.id))
-                            values = {'g_id': gradeable.id, 'gd_overall_comment': 'lorem ipsum lodar'}
+                            values = {'g_id': gradeable.id}
```

Comment:
Trying a fresh destroy-up on this branch leads to the following error. It looks like a setup sample courses failure

![image](https://user-images.githubusercontent.com/11095297/79526339-0b548f80-8033-11ea-9722-67bdfc39bb5b.png)

Reply:
Pushed a fix.